### PR TITLE
Add readme, requirements.txt

### PR DESCRIPTION
**Pull request by akshatvermavi**
Readme.md, requirement.txt is added now user can easy setup the project locally and run and able to work over it

### CODE REVIEW

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
Readme is good, please add the requirements file itself as mentioned in the issue description after  understanding the python code

**Comment by akshatvermavi** [comment]
requirements file itself as mentioned in the issue description after
understanding the python code is now added after careful reading the python
code sir now you can check and review my contributions

**Comment by PriyavKaneria** [comment]
Any updates on this? The requirements.txt file is not yet pushed

**Comment by akshatvermavi** [comment]
Sir I created that txt file and pushed it as well but may be some technical
issue it's not showing up so I am going tho push once again by evening
after my classes

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,83 @@
+attrs==22.2.0
```
Comment (PriyavKaneria) [inline requirements.txt]: It should be in api folder

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
Shift file to api folder and it should be done

**Comment by akshatvermavi** [comment]
Sir now is this in the right position?

Screenshot from 2023-10-14 19-58-07

**Review by PriyavKaneria — APPROVED** [review]
Lgtm

**Comment by PriyavKaneria** [comment]
Fixes #1

**Pull request merged**